### DISCUSSION TITLE
Importing IIS Site CKL creates `servername-Web Site Name` asset

### DISCUSSION THREAD

**Discussion by vvincevv** [NUWCDIVNPT/stig-manager#956] [General]
Hello, I imported several CKLs and noticed any IIS Site CKLs create a new asset with the web site name appended, so I end up with
server-name
server-name-Default Web Site-NA

the server-name asset has several STIGs assigned to it as expected and the other with the 'Default Web Site' suffix has just the 1  IIS Site stig associated to it.

Is that by design? Is here an option to have the iis site stig stay under the server-name asset?

**Comment by cd-rite** [comment]
Hi @vvincevv 
Yep, that is by design! We describe our approach to .ckl imports and Asset Names here: https://stig-manager.readthedocs.io/en/latest/installation-and-setup/data-and-permissions.html#processing-ckl-files
**If/When you export that Asset as a .ckl file, it will have the original Asset name listed.** 

Since certain STIGs apply to instances of  "Assets" can host multiple databases and/or websites that must be evaluated separately, we took the approach of promoting that info to a unique Asset name as stored in STIG Manager to make it easier to navigate.

The only way to prevent that from happening is by removing the specific DB/Website info from the .ckl file you are importing.
